lib/lib/components/modal-pdp-banner.js

```diff
@@ -10,7 +10,7 @@ var _charges = _interopRequireDefault(require("../charges"));
 function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
 var modalPdpBanner = function modalPdpBanner(url) {
-  var iframe = "<div className=\"iframe-container\"><iframe id=\"creditkey-pdp-iframe\" src=\"" + url + "\"></iframe></div>";
+  var iframe = "<div className=\"iframe-container\"><iframe scrolling=\"no\" id=\"creditkey-pdp-iframe\" src=\"" + url + "\"></iframe></div>";
```

Comment:
were we seeing scrollbars under certain conditions?

Reply:
Yes. Chrome and Edge on Windows machines.